tests/framework/umbra_client_allscales.c

```diff
@@ -45,10 +45,13 @@
                 __FILE__, __LINE__, #cond, msg), dr_abort(), 0)))
 
 static void
-test_umbra_mapping(client_id_t id, umbra_map_scale_t scale, const char *label)
+test_umbra_mapping(client_id_t id, umbra_map_scale_t scale, const char *label,
+                   const int scale_val, const bool is_scale_down)
```

Comment:
Not a fan of marking int and bool parameters as const since it has no impact on the caller or interface.  I guess our style guide doesn't say anything, but we're not doing this anywhere else, so for consistency I would suggest removing.  Maybe we should add it to the style guide.

Reply:
I added it to the style page.

Reply:
Done